### ISSUE TITLE
[ test_advanced_reboot ] | FAILED:dut:DUT hasn't shutdown in 300 seconds

### ISSUE THREAD

**Issue by slutati1536** [sonic-net/sonic-mgmt#7926]
**Description**

fast/warm reboot test fail over PTF test failing and not preforming reboot
PR #7817 broke the ansible/roles/test/files/ptftests/advanced-reboot.py file.

**Steps to reproduce the issue:**
1. RUN platform_tests/test_advanced_reboot.py::test_fast_reboot OR platform_tests/test_advanced_reboot.py::test_warm_reboot

**Describe the results you received:**
test will fail over  FAILED:dut:DUT hasn't shutdown in 300 seconds,
but in fact ptf test if failing to preform any reboot on DUT

from log:
**Describe the results you expected:**
PTF test should be preforming reboot on dut

**Additional information you deem important:**

**Output of `show version`:**
```
SONiC Software Version: SONiC.202211_RC8.5-9c3b69550_Internal
Distribution: Debian 11.6
Kernel: 5.10.0-18-2-amd64
Build commit: 9c3b69550
Build date: Tue Mar 28 20:37:18 UTC 2023
Built by: sw-r2d2-bot@r-build-sonic-ci03-242

Platform: x86_64-mlnx_msn3800-r0
HwSKU: Mellanox-SN3800-D112C8
ASIC: mellanox
ASIC Count: 1
Serial Number: MT1925X00008
Model Number: MSN3800-CS2F
Hardware Revision: A2
Uptime: 18:43:08 up  4:56,  1 user,  load average: 0.74, 0.76, 0.89
Date: Thu 30 Mar 2023 18:43:08

Docker images:
REPOSITORY                                         TAG                               IMAGE ID       SIZE
docker-platform-monitor                            202211_RC8.5-9c3b69550_Internal   5b63d28b2986   942MB
docker-platform-monitor                            latest                            5b63d28b2986   942MB
docker-syncd-mlnx                                  202211_RC8.5-9c3b69550_Internal   4729e9c29671   939MB
docker-syncd-mlnx                                  latest                            4729e9c29671   939MB
docker-orchagent                                   202211_RC8.5-9c3b69550_Internal   ae71e16bb899   539MB
docker-orchagent                                   latest                            ae71e16bb899   539MB
docker-fpm-frr                                     202211_RC8.5-9c3b69550_Internal   692b768f92bd   550MB
docker-fpm-frr                                     latest                            692b768f92bd   550MB
docker-teamd                                       202211_RC8.5-9c3b69550_Internal   99204c057be7   520MB
docker-teamd                                       latest                            99204c057be7   520MB
docker-macsec                                      latest                            2eea3568198b   522MB
docker-nat                                         202211_RC8.5-9c3b69550_Internal   6817fd37c0d6   490MB
docker-nat                                         latest                            6817fd37c0d6   490MB
docker-sflow                                       202211_RC8.5-9c3b69550_Internal   9ca511aa2608   488MB
docker-sflow                                       latest                            9ca511aa2608   488MB
docker-sonic-telemetry                             202211_RC8.5-9c3b69550_Internal   be9f9e750657   802MB
docker-sonic-telemetry                             latest                            be9f9e750657   802MB
docker-snmp                                        202211_RC8.5-9c3b69550_Internal   62432cf85f40   549MB
docker-snmp                                        latest                            62432cf85f40   549MB
docker-eventd                                      202211_RC8.5-9c3b69550_Internal   855b7c3d7218   503MB
docker-eventd                                      latest                            855b7c3d7218   503MB
docker-dhcp-relay                                  latest                            c90d1c5967eb   513MB
docker-lldp                                        202211_RC8.5-9c3b69550_Internal   e33be345d3f2   546MB
docker-lldp                                        latest                            e33be345d3f2   546MB
docker-mux                                         202211_RC8.5-9c3b69550_Internal   e9f6fa7ef277   552MB
docker-mux                                         latest                            e9f6fa7ef277   552MB
docker-database                                    202211_RC8.5-9c3b69550_Internal   9cae58961a25   503MB
docker-database                                    latest                            9cae58961a25   503MB
docker-sonic-p4rt                                  202211_RC8.5-9c3b69550_Internal   77b20e5b758d   586MB
docker-sonic-p4rt                                  latest                            77b20e5b758d   586MB
docker-router-advertiser                           202211_RC8.5-9c3b69550_Internal   f3a99688e9df   503MB
docker-router-advertiser                           latest                            f3a99688e9df   503MB
docker-sonic-mgmt-framework                        202211_RC8.5-9c3b69550_Internal   20cfc12ab3c5   612MB
docker-sonic-mgmt-framework                        latest                            20cfc12ab3c5   612MB

```
**Attach debug file `sudo generate_dump`:**
log of PTF test:
[advanced-reboot.ReloadTest.log](https://github.com/sonic-net/sonic-mgmt/files/11113379/advanced-reboot.ReloadTest.log)

**Comment by bingwang-ms** [comment]
Did you look into the reason the DUT is not rebooted? Is it because of connection failure or running incorrect command?

**Comment by yxieca** [comment]
Is this another python3 issue?

**Comment by wsycqyz** [comment]
> 

working on it

**Comment by wsycqyz** [comment]
> Is this another python3 issue?

Working on this

**Comment by wsycqyz** [comment]
Fixed in #8108
